chaoslib/configuration.py

```diff
@@ -53,7 +53,7 @@ def load_configuration(config_info: Dict[str, str]) -> Configuration:
             if value["type"] == "env":
                 env_key = value["key"]
                 env_default = value.get("default")
-                if env_key not in env and not env_default:
+                if env_key not in env and env_default is None:
```

Comment:
I usually advise against parenthesis but, in this case, I might suggest we have them to avoid precdence issue:

```suggestion
                if (env_key not in env) and (env_default is None):
```